bundles/io.openliberty.tools.eclipse.ui/src/io/openliberty/tools/eclipse/DevModeOperations.java

```diff
@@ -189,10 +189,10 @@ public void start(IProject iProject, String parms, String javaHomePath, String m
             String cmd = "";
             BuildType buildType = project.getBuildType();
             if (buildType == Project.BuildType.MAVEN) {
-                cmd = CommandBuilder.getMavenCommandLine(projectPath,
```

Comment:
I believe the same changes should be done in startInContainer.

Reply:
Done